src/arrayMethodSort.js

```diff
@@ -4,8 +4,43 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultCompare = (can1, can2) => {
+    const can1Str = typeof can1 === 'string' ? can1 : can1.toString();
+    const can2Str = typeof can2 === 'string' ? can2 : can2.toString();
+    let currentDif;
+
+    for (let i = 0; i < can1Str.length; i++) {
+      currentDif = can1Str.charCodeAt(i) - can2Str.charCodeAt(i);
+
+      if (currentDif > 0) {
+        return 1;
+      } else if (currentDif === 0) {
+        continue;
+      } else {
+        return -1;
+      }
+    }
+  };
```

Comment:
i think this can be way easier - simply compare strings
```
if (String(a) > String(b)) { return ?
if (String(a) < String(b)) { return ?
return ?
```

Reply:
I forgot about the default comparisons of these level libraries.  Correcting